impeller/display_list/canvas.cc

```diff
@@ -786,6 +786,7 @@ void Canvas::DrawVertices(const std::shared_ptr<VerticesGeometry>& vertices,
     contents->SetEffectTransform(effect_transform);
     contents->SetTexture(texture);
     contents->SetTileMode(x_tile_mode, y_tile_mode);
+    contents->SetSamplerDescriptor(sampler_descriptor);
```

Comment:
for whatever reason, with this change clangd highlighted that `sampler_descriptor` was unused, revealing this bug.